corehq/form_processor/submission_process_tracker.py

```diff
@@ -0,0 +1,69 @@
+# coding=utf-8
+from __future__ import absolute_import
+
+from __future__ import unicode_literals
+import contextlib
+import datetime
+
+from couchforms.models import UnfinishedSubmissionStub, UnfinishedArchiveStub
+
+
+class SubmissionProcessTracker(object):
+    def __init__(self, stub=None):
+        self.stub = stub
+
+    def submission_saved(self):
+        if self.stub:
+            self.stub.saved = True
+            self.stub.save()
+
+    def submission_fully_processed(self):
+        if self.stub:
+            self.stub.delete()
+
+
+class ArchiveProcessTracker(object):
+    def __init__(self, stub=None):
+        self.stub = stub
+
+    def delete_me_get_history(self):
```

Comment:
what's this for?

Reply:
Oops, did not mean to check it in: https://github.com/dimagi/commcare-hq/pull/22470/commits/75b23b43fb7b8aafe388c51f447ebb2ba9df7e04